### PR TITLE
Resolve: "Set Panix Version in Makefile"

### LINKED ISSUE #185: Define Panix Version in Makefile
The current Panix version should be defined in the master Makefile and then set in `CFLAGS` just like the current commit.

### PR DESCRIPTION
**Pull request by Kfeavel**
Closes #185

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,11 @@ export PROJ_NAME = panix
 # Makefile flags
 # prevent make from showing "entering/leaving directory" messages
 export MAKEFLAGS   += --no-print-directory
-export GIT_VERSION := "$(shell git describe --abbrev=8 --dirty --always --tags)"
+# Panix Version
+export GIT_COMMIT := "$(shell git describe --abbrev=8 --dirty --always --tags)"
+export VER_MAJOR := "0"
+export VER_MINOR := "3"
```
Comment (micahswitzer) [inline Makefile]: Are we bumping version numbers right before, or right after release? It looks like we're currently bumping right before, but I do really like doing it immediately after.

**Review by micahswitzer — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (Kfeavel) [reply Makefile]: The current version is 0.3.0 and we'll bump it to 0.4.0 after this December release.

Comment (Kfeavel) [reply Makefile]: Bumped.

Comment (micahswitzer) [reply Makefile]: We're going to start bumping versions following releases.

**Review by micahswitzer — APPROVED** [review]

**Pull request merged**